btag.py

```diff
@@ -40,10 +40,14 @@ class bTagCorrProducer:
     tagger_to_brag_branch = {"particleNet": "PNetB",
                              "deepJet": "DeepFlavB"}
 
-    def __init__(self, period, loadEfficiency=False, tagger_name="deepJet", use_split_jes=False):
+    channel_to_presel_jet_name = {"hh_bbww": "Jet_sel",
+                                  "hh_bbtautau": "Jet_bCand"}
```

Comment:
There should be no analysis-dependent definition in Corrections. Please, implement the column name as a config parameter